### PR TITLE
[Docs] Fix Doxygen build warnings

### PR DESCRIPTION
**Pull request by foundrytom**
#104 should now pass!

### CODE REVIEW

File: python/openassetio/managerAPI/ManagerInterface.py [hunk 1]
```diff
@@ -860,6 +893,15 @@ def finalizedEntityVersion(self, entityRefs, context, hostSession, overrideVersi
         If versioning is unsupported for a given @ref
         entity_reference, then the input reference should be returned.
 
+        @param entityRefs List[str] The entity references to finalize.
```
Comment (feltech) [inline python/openassetio/managerAPI/ManagerInterface.py:896]: [consider]
```suggestion
        @param entityRefs `List[str]` The entity references to finalize.
```

File: python/openassetio/managerAPI/ManagerInterface.py [hunk 2]
```diff
@@ -998,27 +1047,42 @@ def setRelatedReferences(
...
-        one.
+        defining relationships between existing assets (such as
+        connecting the script used to define a render, with the image
+        sequences it creates) and 'register' as being defining the
+        relationship between a new asset and some existing one.
 
         In systems that don't support post-creation adjustment of
         relationships, this can simply be a no-op.
 
+        @param entityRef str, The entity to which the relationship
```
Comment (feltech) [inline python/openassetio/managerAPI/ManagerInterface.py:1065]: [consider] Would be good to be consistent with the `@param` style, at least within a given file unless/until we decide on a global style
```suggestion
        @param entityRef `str` The entity to which the relationship
```

**Review by feltech — CHANGES_REQUESTED** [review]

File: python/openassetio/_core/audit.py [hunk 3]
```diff
@@ -100,6 +100,9 @@ def auditApiCall(group=None, static=False):
 
     @param group str, an optional group name to log the call under @ref
     Auditor.addMethod
+
+    @param static bool, set to True if the decorated function is a
```
[the remark this answers is not in the document]
Comment (foundrytom) [reply python/openassetio/_core/audit.py:104]: I was going for "consistency with local defs" (based on the preceding one for `group`) - basically the same as your comment above.  Given the "It'll all go away in `c++`, shall we stick with local consistency for now?

[the remark this answers is not in the document]
Comment (foundrytom) [reply python/openassetio/_core/audit.py:104]: Yeah, I was leaning towards that, as we should have full explicit typing from the declaration?

**Review by feltech — APPROVED** [review]

**Comment by feltech** [comment]
There's still the [question] of whether to bother with unnecessarily fully-qualified cross-references, but that shouldn't prevent this being merged. So squash and merge when ready.

**Comment by foundrytom** [comment]
> There's still the [question] of whether to bother with unnecessarily fully-qualified cross-references, but that shouldn't prevent this being merged. So squash and merge when ready.

Thanks, yeah, I think we should have `[plan]` via #82 that probably makes sense to do after `c++` - not sure if some of the ambiguity will go away when we don't have the `namespace`/`class` overlap + hoisting fun going on?

**Comment by foundrytom** [comment]
I've made the `Tests / docs (pull_request)` check `Required` now too.

**Pull request merged**